WindowInsetsAnimation/app/src/main/java/com/google/android/samples/insetsanimation/SimpleImeAnimationController.kt

```diff
@@ -36,6 +37,7 @@ import kotlin.math.roundToInt
  * See [InsetsAnimationLinearLayout] and [InsetsAnimationTouchListener] for examples of how
  * to use this class.
  */
+@RequiresApi(30)
 internal class SimpleImeAnimationController {
     private var insetsAnimationController: WindowInsetsAnimationController? = null
```

Comment:
There is a compat version as well if you want to avoid using the RequireApi()

Reply:
Ah cool, added!